lib/src/event_arrangers/side_event_arranger.dart

```diff
@@ -87,7 +89,11 @@ class SideEventArranger<T extends Object?> extends EventArranger<T> {
           left: slotWidth * (sideEvent.column - 1),
           right: slotWidth * (column - sideEvent.column),
           top: startTime.getTotalMinutes * heightPerMinute,
-          bottom: height - endTime.getTotalMinutes * heightPerMinute,
+          bottom: height -
+              (endTime.getTotalMinutes == 0
```

Comment:
You can make one variable to pass the bottom parameter. Write all condition on that variable